spring-integration-http/src/main/resources/org/springframework/integration/http/config/spring-integration-http-4.2.xsd

```diff
@@ -203,6 +203,24 @@
 							]]></xsd:documentation>
 						</xsd:annotation>
 					</xsd:attribute>
+					<xsd:attribute name="reply-timeout-status-code-expression" type="xsd:string">
+						<xsd:annotation>
+							<xsd:documentation>
+								A SpEL expression that resolves to an 'HttpStatus' code when rendering a response after
+								a 'reply-timeout'.
+								The expression must return an object which can be converted to a
+								'org.springframework.http.HttpStatus' enum value.
+								The 'evaluationContext' has a 'BeanResolver' but no variables, so the usage of this attribute
+								is somewhat limited.
+								An example might be to resolve, at runtime, some scoped Bean that returns an
+								'HttpStatus' value.
+								By default 'status-code-expression' is null, meaning that the default '200 OK' response status
+								will be returned.
+								The 'http:inbound-gateway' resolves the 'status code' from the 'http_statusCode' header of the reply
+								Message.
```

Comment:
This doc is copy/paste from the `status-code-expression`.
Shouldn't it be fixed according to its logic? Especially for the `gateway` case...